src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,121 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private int capacity = DEFAULT_CAPACITY;
+    private Node<K, V>[] table = new Node[capacity];
+    private int size = 0;
+    private int threshold = 0;
```

Comment:
You have a field `threshold` but it seems that you never initialize it. As a result, your resize method will never be called. Don't forget to initialize your `threshold` field with the initial capacity multiplied by the load factor.